dms_attachment_link/views/dms_file_views.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="view_dms_file_kanban_wizard_selection" model="ir.ui.view">
+        <field name="name">dms_file.kanban.wizard.selection.attachment.link</field>
+        <field name="model">dms.file</field>
+        <field name="inherit_id" ref="dms.view_dms_file_kanban_wizard_selection" />
+        <field name="arch" type="xml">
```

Comment:
This view should be primary for having multiple inheritance.

Reply:
I think it is not necessary, the template that is primary is `dms.view_dms_file_kanban_wizard_selection`, and in `dms` there is also `action_dms_file_wizard_selector`. The `dms_attachment_link` module just adds the button to the already existing `dms` view.

Reply:
Exactly, but if you add a new module using this file selector, this button "Add" will appear also, and we don't want that.

Reply:
Ok, changes done.